packages/datadog-plugin-couchbase/src/index.js

```diff
@@ -32,7 +32,8 @@ class CouchBasePlugin extends Plugin {
       tags
     })
 
-    span.setTag('couchbase.bucket.name', bucket.name || bucket._name)
+    if (bucket) span.setTag('couchbase.bucket.name', bucket.name) // semver >=2 <3
+    if (collection) span.setTag('couchbase.collection.name', collection.name) // semver >=3
```

Comment:
Can these be normalized so that the plugin doesn't need to know about the version of the library?